### PR TITLE
add aws-nuke

### PR DESCRIPTION
**Pull request by osterman**
## what
* Add nightly cron to nuke testing account

## why
* When tests bomb they sometimes leave artifacts behind
* `aws-nuke` fixes that

### CODE REVIEW

File: .github/aws-nuke.yaml [hunk 1]
```diff
@@ -0,0 +1,149 @@
+regions:
+#- us-east-1
+- us-east-2
+
+account-blacklist:
+- "999999999999" # production
+
+resource-types:
+  # only nuke these three resources
```
Comment (aknysh) [inline .github/aws-nuke.yaml]:
```suggestion
  # only nuke these resources
```

File: .github/aws-nuke.yaml [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+        value: "^$"
+      EC2RouteTable:
+      - property: "tag:Name"
+        type: "regex"
+        value: "^$"
+      EC2Subnet:
+      - property: "DefaultForAz"
+        value: "true"
+      EC2VPC:
+      - property: "IsDefault"
```
Comment (aknysh) [inline .github/aws-nuke.yaml:70]: not sure how it works, but this (and the value: "true") will not nuke the default VPC (which we should not nuke)?

Comment (osterman) [reply .github/aws-nuke.yaml:70]: Ya, I'm ignore the default VPC. the `filters` section is everything to ignore.

Comment (osterman) [reply .github/aws-nuke.yaml:70]: (and below `cpco-*` is always ignored)

Comment (osterman) [reply .github/aws-nuke.yaml:70]: and everything else that's not the default or cpco (e.g. `eg-*` is nuked)

**Review by aknysh — APPROVED** [review]

**Pull request merged**